usecase/usecase-webapi-xwalk-tests/testscripts/steps/steps.py

```diff
@@ -47,6 +47,14 @@ def check_type_by_key(context, key, expecttype):
 def check_type_by_key(context, p_name, key):
     assert context.app.save_content(p_name, key)
 
+@step(u'I commit message by click "{key}"')
+def i_click(context, key):
+    assert context.app.click_element_by_id_with_js(key)
+
+@step(u'I fill in element "{key}" by "{attr}" with "{text}"')
+def fill_with_text(context, key, attr, text):
```

Comment:
Please rename this function name, otherwise it will cause repetition when remove to ATIP.